Description: Stop unnecessary linkage against libgpg-error.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Last-Update: 2014-07-03

--- a/lib/m4/hooks.m4
+++ b/lib/m4/hooks.m4
@@ -51,7 +51,7 @@ AC_DEFUN([LIBGNUTLS_HOOKS],
     if test "$libgcrypt" = "yes"; then
         cryptolib=libgcrypt
         AC_DEFINE([HAVE_GCRYPT], 1, [whether the gcrypt library is in use])
-	AC_LIB_HAVE_LINKFLAGS([gcrypt], [gpg-error], [#include <gcrypt.h>],
+	AC_LIB_HAVE_LINKFLAGS([gcrypt], , [#include <gcrypt.h>],
                       [enum gcry_cipher_algos i = GCRY_CIPHER_CAMELLIA128])
       if test "$ac_cv_libgcrypt" != yes; then
         AC_MSG_ERROR([[
